modules/nf-core/kaiju/mergeoutputs/main.nf

```diff
@@ -0,0 +1,51 @@
+process KAIJU_MERGEOUTPUTS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kaiju:1.10.0--h43eeafb_0':
+        'biocontainers/kaiju:1.10.0--h43eeafb_0' }"
+
+    input:
+    tuple val(meta), path(kaiju), path(kraken)
+    path (db)
+
+    output:
+    tuple val(meta), path("*_merged.tsv"), emit: merged
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args ?: ''
+    def prefix  = task.ext.prefix ?: "${meta.id}"
+    def dbnodes = db ? '-t <(find -L ${db} -name "*nodes.dmp")' : ''
+    """
+    kaiju-mergeOutputs \\
+        -i <(sort -k2,2 ${kaiju}) \\
```

Comment:
Maybe also add the possibility to supply arguments to `sort`?

Reply:
Kaiju mergeoutputs only works if it's sorted based on the read ID. Providing args here will provide very little but the module can then easily be broken. So I'm not certain if it's necessary.

Reply:
Okay good point, ignore me here :wink: